packages/docusaurus-theme-classic/src/theme/DocLegacyItem/styles.module.css

```diff
@@ -23,3 +23,30 @@
     display: none;
   }
 }
+
+.editButton {
```

Comment:
I've removed all of these because we should be using Infima classes instead. Anyway I decided to make it look like a link instead of a button since it's the more common practice for docs.